README.md

```diff
@@ -82,7 +84,7 @@ $ brew install carthage
 To integrate PinpointKit into your Xcode project using Carthage, specify it in your `Cartfile`:
 
 ```ogdl
-github "Lickability/PinpointKit" ~> 0.9
+github "Lickability/PinpointKit" ~> "swift-2.3"
```

Comment:
I get `Parse error: unexpected trailing characters in line: github "Lickability/PinpointKit" ~> "swift-2.3"`

I think this needs to be `github "Lickability/PinpointKit" "swift-2.3"`